File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.js
/**
 * @param {string} s
 * @return {string}
 */
var removeOuterParentheses = function(s) {

    let level = 0
    let ans = "";
    for(let i=0;i<s.length;i++){
        if(s[i]==="("){
            ++level;
            ans += ((level > 1)? s[i] : "");
            
        }else{
            ans += ((level > 1)? s[i] : "");
            --level;
        }
    }
    return ans;
    
};

File: 1078-remove-outermost-parentheses/remove-outermost-parentheses.java
class Solution {
    public String removeOuterParentheses(String s) {
        int level = 0;
        String ans ="";

        for(int i=0;i<s.length();i++){
            if(s.charAt(i)=='('){
                ++level;
                ans += ((level > 1)?s.charAt(i):"");
            }else{
                ans += ((level > 1)?s.charAt(i):"");
                --level;
            }
        }
        return ans;
        
    }
}